### DISCUSSION TITLE
Replace our textsanitizer with s9e/TextFormatter or similar lib

### DISCUSSION THREAD

**Discussion by MekDrop** [ImpressCMS/impresscms#1035] [Ideas]
I think to save some time dealing with bbcodes we could use some composer lib. 

[s9e/TextFormatter](https://github.com/s9e/TextFormatter) looks good. It's classes based so it would be possible to write modules that could extend shortcodes functionality.

**Comment by thunderer** [comment]
Adding a comment as per our discussion in ImpressCMS/impresscms#361: would you like to evaluate a new BBCode processing library? Please let me know what do you think about this one: https://github.com/thunderer/Shortcode. I'll be happy to answer your questions and talk about feature requests if you have any.
